packages/extension-ui/src/Popup/ImportSeed/SeedAndPath.tsx

```diff
@@ -45,7 +45,7 @@ function SeedAndPath ({ className, onAccountChange, onNextStep }: Props): React.
     validateSeed(suri).then((newAccount) => {
       setError('');
       setAddress(newAccount.address);
-      onAccountChange({ ...newAccount });
+      onAccountChange(newAccount);
```

Comment:
What I would need here, is `{ ...newAccount, genesis }` to take the genesis into account. but for some reason this breaks the tests, even `{ ...newAccount}` breaks the tests.. although it's the same as `newAccount` which passes fine..

Reply:
Giving up on that using Object.assign instead of the spread operator.
It looks like we're using all the relevant `@babel/plugin-proposal-object-rest-spread` from https://github.com/polkadot-js/dev/blob/master/packages/dev/config/babel-plugins.cjs still, jest hangs, and doesn't even spit out errors.